core/templates/dev/head/pages/admin/admin.html

```diff
@@ -135,6 +135,38 @@
     <script src="/templates/dev/head/domain/user/UserInfoObjectFactory.js"></script>
     <script src="/templates/dev/head/domain/utilities/UrlInterpolationService.js"></script>
 
+    <script src="/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.module.js"></script>
+    <script src="/templates/dev/head/pages/learner-dashboard-page/learner-dashboard-page.controller.js"></script>
```

Comment:
Could you please add some comments explaining what the scripts are for?
like:
```
<!--Learner Dashboard page scripts-->
```

Reply:
done